From 30f9665d6ed8869af2299239f41b9764f558f9c5 Mon Sep 17 00:00:00 2001
From: oszilloskop <oszilloskop@users.noreply.github.com>
Date: Thu, 7 Feb 2019 01:55:07 +0100
Subject: [PATCH] ar71xx: fix incorrect gl-ar150 and gl-ar300m image names
 (#1640)

also added manifest aliases for gl-ar150, gl-ar300m, gl-ar750
---
 targets/ar71xx-generic | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/targets/ar71xx-generic b/targets/ar71xx-generic
index 2f6828ba..bd028dfa 100644
--- a/targets/ar71xx-generic
+++ b/targets/ar71xx-generic
@@ -72,14 +72,19 @@ factory
 # GL Innovations
 
 device gl-inet-6408a-v1 gl-inet-6408A-v1
+
 device gl-inet-6416a-v1 gl-inet-6416A-v1
 
-device gl-ar150 gl-ar150
+device gl.inet-gl-ar150 gl-ar150
+manifest_alias gl-ar150
 factory
-device gl-ar300m gl-ar300m
+
+device gl.inet-gl-ar300m gl-ar300m
+manifest_alias gl-ar300m
 factory
 
 device gl.inet-gl-ar750 gl-ar750
+manifest_alias gl-ar750
 packages $ATH10K_PACKAGES
 factory
 
-- 
2.17.1

